src/Dev/DevelopmentAdmin.php

```diff
@@ -104,6 +97,31 @@ public function runRegisteredController(HTTPRequest $request)
         }
     }
 
+    public function canInit(): bool
+    {
+        return (
+            Director::isDev()
+            // We need to ensure that DevelopmentAdminTest can simulate permission failures when running
+            // "dev/tasks" from CLI.
+            || (Director::is_cli() && RootDevelopmentAdmin::config()->get('allow_all_cli'))
+            || Permission::check("ADMIN")
+            || Permission::check("ALL_DEV_ADMIN")
+            || Permission::check("CAN_DEV_GRAPHQL")
+        );
+    }
+
+    public function providePermissions(): array
+    {
+        return [
+            'CAN_DEV_GRAPHQL' => [
+                'name' => _t(__CLASS__ . '.CAN_DEV_GRAPHQL_DESCRIPTION', 'Can access GraphQL tools'),
```

Comment:
```suggestion
                'name' => _t(__CLASS__ . '.CAN_DEV_GRAPHQL_DESCRIPTION', 'Can access /dev/graphql'),
```
This is consistent with the wording of permissions in the other PR.